app/src/main/java/com/example/arc_exapmle/user/UserRepository.kt

```diff
@@ -59,36 +73,9 @@ class UserRepository(application: Application) {
 
         }
 
-    }
-
-    private class DeleteUserAsyncTask(userDaoImpl: UserDao) : AsyncTask<UserUI, Unit, Unit>() {
```

Comment:
balakhare az share in async task ha khalas shodim!

Reply:
:)